packages/browser-sdk/src/solana/signAndSendTransaction.ts

```diff
@@ -30,5 +32,9 @@ export async function signAndSendTransaction(
   if (!provider.isConnected) {
     throw new Error("Provider is not connected even after attempting to connect.");
   }
-  return provider.signAndSendTransaction(transaction);
+
+  // Convert the Kit transaction into a web3.js VersionedTransaction before sending.
+  const versionedTransaction: VersionedTransaction = transactionToVersionedTransaction(transaction);
```

Comment:
According to discussions with solana and anza teams, we can create minimal types that only specify the properties that we use in the provider.
Like adding only toBase58 or serialize methods and maybe version field. Essentially we would simulate it's a web3.js object.

Reply:
That would let us remove the web3.js dependency